README.md

```diff
@@ -53,6 +53,12 @@ Start the server using:
 uvicorn tdp_server.main:app --reload
 ```
 
+## Build Docker Container
+
+```bash
+docker build -t tdp_server -f docker/Dockerfile .
+```
+
```

Comment:
Can you add a section on how to run the container? The same as you did in the description of the PR is good

Reply:
"The command I showed", the `docker build` command ? I agree with @leopaul36 why not a run command ?

Reply:
I added a section about running the container

Reply:
Is that okay with you @rpignolet ?

Reply:
yes